lib/helpers/link_helper.rb

```diff
@@ -14,6 +15,11 @@ def canonical_link
       "#{CANONICAL_ROOT}documentation#{clean_item.gsub("/root", "")}".gsub(%r{/$}, '')
     end
   end
+
+  def image_url(image_path)
+    return image_path if image_path.match(/^http/)
```

Comment:
is the regex case insensitive by default ie `HTTP` or `http`?

Reply:
No, good point too, thanks

Reply:
See 
7439acb